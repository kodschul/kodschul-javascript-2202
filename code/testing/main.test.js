const { sum, createUser } = require("./main");

describe("main Tests", () => {
  it("should add 1 + 2", () => {
    expect(sum(1, 2)).toBe(3);
  });

  it("should create new user", () => {
    const user = createUser("John Doe");
    expect(user).toMatchSnapshot();
    // expect(user).toBe("John Doe1");
    // expect(user.firstName).toBe("John");
    // expect(user.lastName).toBe("Doe");
  });
});
