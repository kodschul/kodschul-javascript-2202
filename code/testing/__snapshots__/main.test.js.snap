// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main Tests should create new user 1`] = `
{
  "firstName": "John",
  "id": "JohnDoe1",
  "lastName": "Doe",
  "name": "John Doe",
}
`;
